gum/backend-darwin/gumobjcapiresolver.c

```diff
@@ -151,6 +161,8 @@ gum_objc_api_resolver_finalize (GObject * object)
 {
   GumObjcApiResolver * self = GUM_OBJC_API_RESOLVER (object);
 
+  g_object_unref (self->monitor);
```

Comment:
References to other objects should be released in `::dispose` instead of `::finalize` — some details [here](https://www.manpagez.com/html/gobject/gobject-2.42.0/howto-gobject-destruction.php).